### PR TITLE
[CUBRIDMAN-91] Improve JDBC driver to use statement cache

### PR DESCRIPTION
**Pull request by airnet73**
http://jira.cubrid.org/browse/CUBRIDMAN-91

Purpose
3 properties have been added to use the statement cache
Each property description and usage examples are required.

Properties
usePreparedStmtCache
preparedStmtCacheSize
preparedStmtCacheSqlLimit

Implementation
N/A

Remarks
N/A

### CODE REVIEW

File: ko/api/jdbc.rst [hunk 1]
```diff
@@ -196,6 +199,10 @@ JDBC 프로그래밍
        *   범위는 1 ~ 1024 (1메가 바이트에서 to 1기가 바이트)
```
Comment (kisoo-han) [inline ko/api/jdbc.rst]: 범위는 1 ~ 1024 (1메가 바이트에서 1기가 바이트)

File: ko/api/jdbc.rst [hunk 2]
```diff
@@ -196,6 +199,10 @@ JDBC 프로그래밍
...
        *   기본 값은 1 (메가 바이트)
 
+    *  **usePreparedStmtCache**: Prepared Statement 캐시 여부 (기본값: false)
+    *  **preparedStmtCacheSize**: usePreparedStmtCache이 TRUE일 경우, 캐싱할 수 있는 갯수  (기본:25, 최소:1, 최대:2147483647)
+    *  **preparedStmtCacheSqlLimit**: usePreparedStmtCache이 TRUE일 경우, 캐싱할 수 있는 SQL의 길이 (기본:256, 최소:1, 최대:2147483647)
```
Comment (kisoo-han) [inline ko/api/jdbc.rst]: usePreparedStmtCache이 -> usePreparedStmtCache가

**Review by kisoo-han — APPROVED** [review]

File: ko/api/jdbc.rst (hunk 2 above)
Comment (airnet73) [reply ko/api/jdbc.rst]: Thank you for your comment.

File: ko/api/jdbc.rst (hunk 1 above)
Comment (airnet73) [reply ko/api/jdbc.rst]: Thank you for your comment.

**Review by charischau — APPROVED** [review]

**Pull request merged**